packages/react-components/src/components/table-output-component.tsx

```diff
@@ -490,11 +500,11 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
         }
     }
 
-    private async findNextMatchIndex(currRowIndex: number) {
+    private async findMatchIndex(currRowIndex: number, direction = Direction.NEXT) {
```

Comment:
This isn't working as expected... Direction.NEXT is actually the number 0 and Direction.PREVIOUS is the number 1. So later the toString() in the map puts "0" or "1"...

Reply:
Direction[direction] seems to return the string